tests/linalg/test_tile_linalg_hessenberg.py

```diff
@@ -69,4 +69,4 @@ def hessenberg_decomposition_fn(x, xsdiag):
 
         start, end = np.asarray(start)[0], np.asarray(end)[0]
         hessenberg_cycle_count = end[0] - start[0]
-        assert hessenberg_cycle_count <= 105000
+        assert hessenberg_cycle_count <= 150000
```

Comment:
Is it a tight bound? (or is the cycle close closer to 100k?) Just to tend to prefer tight bounds on cycle counts as they are accurate measurements on IPU hardware.

Reply:
Unfortunately it is relatively tight. The count was ~144k.